test/sets.jl

```diff
@@ -1,5 +1,9 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+isdefined(Main, :TestHelpers) ||
+                        @eval Main include(joinpath(dirname(@__FILE__), "TestHelpers.jl"))
```

Comment:
This was bugging me too, I have now stolen the style from [arrayops.jl](https://github.com/JuliaLang/julia/blob/master/test/arrayops.jl)